config.yml

```diff
@@ -350,4 +350,5 @@ genesis:
         supplier_unbonding_period_sessions: 1 # Small value to enable for quick testing & onboarding during LocalNet.
         application_unbonding_period_sessions: 1 # Small value to enable for quick testing & onboarding during LocalNet.
         gateway_unbonding_period_sessions: 1 # Small value to enable for quick testing & onboarding during LocalNet.
-        compute_units_to_tokens_multiplier: 42 # Arbitrary value that acts as the answer to everything.
+        compute_units_to_tokens_multiplier: 42000000 # Arbitrary value that acts as the answer to everything.
```

Comment:
Should this still be the big number that it is?

Reply:
Yes, I want to to keep the value of a single compute unit to `42upokt`.

Otherwise we'll have to change all the claim rewards assertions in E2E tests.

Reply:
Yes, I want to to keep the value of a single compute unit to 42upokt.

Otherwise we'll have to change all the claim rewards assertions in E2E tests.